### PR TITLE
Cache accounts - Closes #688

### LINKED ISSUE #688: Cache accounts
### Description

Create accounts cache for SDKv5 account implementation.

The cache should reduce number of requests to Lisk Core.

### Acceptance Criteria

- Use Redis configuration to keep the most often used accounts (LRU-policy).
- The account retrieval from Lisk Core should always update the Redis database.
- Create a different function to retrieve cached account data.

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?
This PR resolves #688 

### How was it solved?
- [x] Use Redis configuration to keep the most often used accounts (LRU-policy)
- [x] Account retrieval from Lisk Core always updates the Redis LRU cache
- [x] Created a different function to retrieve cached account data
- [x] Always fetch account from the cache, if unavailable fetch from Core

### How was it tested?
Local

### CODE REVIEW

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

File: services/core/shared/core/compat/sdk_v5/accounts.js [hunk 1]
```diff
@@ -15,6 +15,7 @@
  */
 const BluebirdPromise = require('bluebird');
 const {
+	CacheLRU,
```
Comment (MichalTuleja) [inline services/core/shared/core/compat/sdk_v5/accounts.js:18]: This LRU-cache implementation doesn't use Redis. Please make it Redis-based.

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

File: services/core/shared/core/compat/sdk_v5/accounts.js [hunk 2]
```diff
@@ -64,6 +64,8 @@ const getAccountsIndex = () => mysqlIndex('accounts', accountsIndexSchema);
 const getBlocksIndex = () => mysqlIndex('blocks', blocksIndexSchema);
 const getTransactionsIndex = () => mysqlIndex('transactions', transactionsIndexSchema);
 
+const ACCOUNTS_CACHE_TTL = 15 * 1000; // in milliseconds
+const accountsCache = CacheRedis('accounts', config.endpoints.redis);
```
Comment (MichalTuleja) [inline services/core/shared/core/compat/sdk_v5/accounts.js:68]:
```suggestion
const accountsCache = CacheRedis('accounts', config.endpoints.volatileRedis);
```

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

**Review by MichalTuleja — APPROVED** [review]
The implementation looks good. Let's perform final tests on continuous delivery.

**Pull request merged**